docs/source/user-guide/vision/VisionData.rst

```diff
@@ -20,22 +27,27 @@ Common Class Parameters
 
 - **task_type** - The task type of the data, can be either ``classification``, ``object_detection``,
   ``semantic_segmentation``, or ``other``. Data format validation is done upon creation of VisionData based
-  on the selected task type. See :doc:`section </user-guide/vision/supported_tasks_and_formats>`
-  for more information.
+  on the selected task type. See the :doc:`supported tasks and formats </user-guide/vision/supported_tasks_and_formats>`
+  section for more information.
 
 - **label_map** - A dictionary mapping class ids to their names.
 
 - **reshuffle_data** - Whether to reshuffle the data. Since data must be shuffled for the checks to
   work properly, only set this to False if you are sure that the data is already shuffled.
 
+To see all other class parameters, see the :class:`VisionData <deepchecks.vision.VisionData>` API reference.
+
 .. _vision_data__creating_vision_data:
```

Comment:
is this label used or should it be deleted? (I see  title referred directly from above in: <#creating-a-visiondata-object>)